functions/src/RSVP/submitRSVP.js

```diff
@@ -0,0 +1,39 @@
+const { db, functions } = require('../utils/firestore');
+const { constraints } = require('./constraints');
+
+exports.submitRSVP = functions.https.onRequest((request, response) => {
+  response.set('Access-Control-Allow-Origin', '*');
+  response.set('Access-Control-Allow-Headers', 'Content-Type, crossDomain');
+
+  const { id } = JSON.parse(request.body);
+
+  const { 
+    emergencyContactName,
+    emergencyContactNumber,
+    tShirtSize,
+    dietRestriction,
+    wouldLikeToSee } = JSON.parse(request.body).data;
+  
+  const rsvpData = {
+    rsvp: {
+      emergencyContactName: emergencyContactName,
+      emergencyContactNumber: emergencyContactNumber,
+      tShirtSize: tShirtSize,
+      dietRestriction: dietRestriction,
+      wouldLikeToSee: wouldLikeToSee,
+    },
+    tags: { 
+      going: true 
+    }
+  }
+
+  const errors = validate(rsvpData.rsvp, constraints.rsvp);
```

Comment:
I think you forgot to import `validate`
https://github.com/nwplus/nwhacks2019/blob/6b6468dba14c6a182639aff43c2cb63e20d5b44d/functions/src/submitVolunteerApplication/index.js#L1